drivers/include/periph/uart.h

```diff
@@ -271,6 +271,31 @@ int uart_mode(uart_t uart, uart_data_bits_t data_bits, uart_parity_t parity,
  */
 void uart_write(uart_t uart, const uint8_t *data, size_t len);
 
+/**
+ * @brief   Write a single byte of data to the specified UART device
+ *
+ * This function is blocking, as it will only return after @p data
+ * has been send.
+ *
+ * @param[in] uart          UART device to use for transmission
+ * @param[in] data          byte to write
+ *
+ */
+void uart_write_byte(uart_t uart, uint8_t data);
```

Comment:
Can't this simply be a static inline around `uart_write`?

Reply:
Sure, just clutters the common header a bit.